libraries/SITL/SITL.h

```diff
@@ -154,6 +154,7 @@ class SITL {
     AP_Float arspd_fail_pitot_pressure; // 1st pitot tube failure pressure
     AP_Float arspd2_fail_pressure; // 2nd pitot tube failure pressure
     AP_Float arspd2_fail_pitot_pressure; // 2nd pitot tube failure pressure
+    AP_Int16 arspd_offset[2]; // airspeed sensor offsets
```

Comment:
These aren't currently used in the autotest suite.  Logically they belong in a separate PR, but if you can say they're tested and working in SITL that's OK.

I'd point out again this PR: https://github.com/ArduPilot/ardupilot/pull/13111 which introduces a variety of airspeed faults.  I will be getting that in at some stage...

Reply:
moved the relevant commit to the EKF3 lane switching and affinity PR